python/simpler/worker.py

```
@@ -1235,6 +1374,429 @@ def _comm_plan_rootinfo_path(self) -> str:
+        return {
+            "session_id": int(session_id),
+            "parent_worker_level": int(self.level),
+            "remote_worker_level": 3,
+            "endpoint_id": int(endpoint_id),
+            "platform": spec.platform,
+            "runtime": spec.runtime,
+            "device_ids": list(spec.device_ids),
+            "num_sub_workers": int(spec.num_sub_workers),
+            "heap_ring_size": self._config.get("remote_heap_ring_size", None),
+            "transport": spec.transport,
+            "listen_host": "127.0.0.1" if loopback else "0.0.0.0",
```

Ruff (0.15.17)

[error] Possible binding to all interfaces

(S104)